queries.sql

```diff
@@ -0,0 +1,27 @@
+/* Querries on the 'animals' table of 'vet_clinic' */
+
+-- Find all animals whoes name ends in "mon"
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] Kindly notice that you should select the animals that also were born on 2019 so your date should be until `2019-12-31`.

```sql
SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-13';
```